src/azure-cli/azure/cli/command_modules/acs/_consts.py

```diff
@@ -206,6 +206,23 @@
 CONST_PRIVATE_DNS_ZONE_CONTRIBUTOR_ROLE = "Private DNS Zone Contributor"
 CONST_DNS_ZONE_CONTRIBUTOR_ROLE = "DNS Zone Contributor"
 
+CONST_CUSTOM_CA_TEST_CERT = '-----BEGIN CERTIFICATE-----\n' \
```

Comment:
It seems this is only used in test? If so, please move it to somewhere under `tests/latest` and it won't be included in the release